ReactWindows/ReactNative.Shared/Bridge/FileJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Threading.Tasks;
+#if WINDOWS_UWP
+using Windows.Storage;
+#else
+using System.IO;
+using System.Reflection;
+#endif
+
+namespace ReactNative.Bridge
+{
+    class FileJavaScriptBundleLoader : JavaScriptBundleLoader
+    {
+        private string _script;
+
+        public FileJavaScriptBundleLoader(string fileName)
+        {
+            SourceUrl = fileName;
+        }
+
+        public override string SourceUrl
+        {
+            get;
+        }
+
+        public override async Task InitializeAsync()
+        {
+            try
+            {
+#if WINDOWS_UWP
+                var storageFile = await StorageFile.GetFileFromApplicationUriAsync(new Uri(SourceUrl)).AsTask().ConfigureAwait(false);
+                _script = storageFile.Path;
+#else
+                var assembly = Assembly.GetAssembly(typeof(FileJavaScriptBundleLoader));
+                var assemblyName = assembly.GetName();
+                var pathToAssembly = Path.GetDirectoryName(assemblyName.CodeBase);
+                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl.Replace("ms-appx:///", String.Empty));
```

Comment:
>(" [](start = 91, length = 2)

I actually don't really like this approach.  Why would you use "ms-appx:///" as a protocol for Windows 7?  Each platform gets to define it's own URI protocol.  Can't you just use the [file:///](file:///) for WPF?